pantheon-files-daemon/FileManager1.vala

```diff
@@ -43,7 +43,8 @@ public class FileManager1 : Object {
         string cmd = "pantheon-files -t";
 
         foreach (string s in uris) {
-            cmd += (" " + s);
+            s = s.replace ("'", "%27").replace ("%", "%%");
+            cmd += (" \'" + PF.FileUtils.sanitize_path (s, null, false) + "\'");
```

Comment:
Is it necessary to to this before PF.FileUtils.sanitize_path?.   If you do it after, do you need to have the extra parameter stopping the unescaping?  See my PR